music_detection/utils/StaffRemover.py

```diff
@@ -93,4 +93,4 @@ def staffDetection(img: np.ndarray, removeLines: bool = True) -> Tuple[List[np.n
     staff_crop = cropImage(img, splittingLines)
```

Comment:
There's a small mistake on my part here : just for the sake of exactness, the case where removeLines==false has to be handled as well (im_morph is croped instead of lines_removed)

Reply:
Make a commit about it